deployment/ansible/group_vars/test

```diff
@@ -22,7 +22,7 @@ celery_processes_per_worker: 1
 itsi_base_url: "https://learn.staging.concord.org/"
 itsi_secret_key: "{{ lookup('env', 'MMW_ITSI_SECRET_KEY') }}"
 
-tilecache_bucket_name: "tilecache.mmw-dev.azavea.com"
+tilecache_bucket_name: "tile-cache.staging.app.wikiwatershed.org"
```

Comment:
In reality this is dynamic based on overall stack domain, so tile cache buckets for staging and production are separate.